src/Transition/CreateNewIDFUsingRulesV9_4_0.f90

```diff
@@ -126,6 +126,7 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
   CHARACTER(len=MaxNameLength) :: OutputDiagnosticsName
   CHARACTER(len=MaxNameLength), ALLOCATABLE, DIMENSION(:) :: OutputDiagnosticsNames
   LOGICAL :: alreadyProcessedOneOutputDiagnostic=.false.
+  INTEGER :: nE, nEC, nG, nNG, nFO, nFON
```

Comment:
Minor detail - I think these could be declared in the Subroutine instead of here, but if no other changes are needed, and it's working, then OK.

Reply:
Just moved and pushed again.